pallets/loans/src/lib.rs

```diff
@@ -629,6 +629,31 @@ pub mod pallet {
             Ok(().into())
         }
 
+        /// Sets a new liquidation incentive percentage for `currency_id`.
+        ///
+        /// Does nothing if the provided asset is not attached to an existent incentive.
+        ///
+        /// - `currency_id`: the asset that is going to be modified.
+        #[pallet::weight(T::WeightInfo::repay_borrow_all())]
+        #[transactional]
+        pub fn set_liquidation_incentive(
+            origin: OriginFor<T>,
+            currency_id: CurrencyId,
+            liquidate_incentive: Ratio,
+        ) -> DispatchResultWithPostInfo {
+            ensure_root(origin)?;
+            Self::ensure_currency(&currency_id)?;
+
+            let does_not_have_incentive = <LiquidationIncentive<T>>::try_get(currency_id).is_err();
```

Comment:
Better to have a default value for `LiquidationIncentive` storage.

Reply:
A default value for the genesis configuration?